src/components/ProfilePage/ProfilePage.js

```diff
@@ -0,0 +1,84 @@
+import './profilePage.scss';
+import default_user from '../../assets/images/default-user.jpg';
+import Avatar from './Avatar/Avatar';
+import {useEffect, useState, useRef} from 'react';
+import $ from 'jquery';
+import React from 'react';
+const ProfilePage = (props) =>{
```

Comment:
Please, fix code formatting.

Reply:
What's the purpose of using functional components?

Reply:
What is the purpose of using jquery?